<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>КДС ХИТ</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBoxCOMPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>COM Порт:</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBoxCOMPort"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnCOMPortConnect">
        <property name="text">
         <string>Подключить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCOMPortDisconnect">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Отключить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxDiagnosticMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Режим диагностики</string>
    </property>
    <widget class="QRadioButton" name="rbModeDiagnosticAuto">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Автоматический режим</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbModeDiagnosticManual">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ручной режим</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>571</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabEventLog">
     <attribute name="title">
      <string>Журнал событий</string>
     </attribute>
     <widget class="QPlainTextEdit" name="EventLog">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>561</width>
        <height>441</height>
       </rect>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEventLogClear">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>561</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Очистить журнал</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabVoltageOnTheHousing">
     <attribute name="title">
      <string>Напряжение на копрусе</string>
     </attribute>
     <widget class="QLabel" name="labelVoltageOnTheHousing2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>VoltageOnTheHousing2</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelVoltageOnTheHousing1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>VoltageOnTheHousing1</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabInsulationResistance">
     <attribute name="title">
      <string>Сопротивление изоляции</string>
     </attribute>
     <widget class="QLabel" name="labelInsulationResistance4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>InsulationResistance4</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelInsulationResistance1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>InsulationResistance1</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelInsulationResistance2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>InsulationResistance2</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelInsulationResistance3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>InsulationResistance3</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabOpenCircuitVoltageGroup">
     <attribute name="title">
      <string>Напряжение разомкнутой цепи группы</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabClosedCircuitVoltageGroup">
     <attribute name="title">
      <string>Напряжение замкнутой цепи группы</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabClosedCircuitVoltageBattery">
     <attribute name="title">
      <string>Напряжение замкнутой цепи батареи</string>
     </attribute>
     <widget class="QLabel" name="labelClosedCircuitVoltage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ClosedCircuitVoltage</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabInsulationResistanceMeasuringBoardUUTBB">
     <attribute name="title">
      <string>Сопротивление изоляции платы измерительной УУТББ</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabOpenCircuitVoltagePowerSupply">
     <attribute name="title">
      <string>Напряжение разомкнутой цепи блока питания</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabClosedCircuitVoltagePowerSupply">
     <attribute name="title">
      <string>Напряжение замкнутой цепи блока питания</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnStartAutoModeDiagnostic">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPauseAutoModeDiagnostic">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>470</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Пауза</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBuildReport">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>470</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сформировать файл-отчет</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxCheckParams">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>471</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Проверяемый параметр</string>
    </property>
    <widget class="QRadioButton" name="rbVoltageOnTheHousing">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Напряжение на корпусе</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbInsulationResistance">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>301</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Сопротивление изоляции</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbOpenCircuitVoltageGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>301</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Напряжение разомкнутой цепи группы</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbClosedCircuitVoltageGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>301</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Напряжение замкнутой цепи группы</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbClosedCircuitVoltageBattery">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>301</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Напряжение замкнутой цепи батареи</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbInsulationResistanceMeasuringBoardUUTBB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>301</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Сопротивление изоляции платы измерительной УУТББ</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbOpenCircuitVoltagePowerSupply">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>301</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Напряжение разомкнутой цепи блока питания</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbClosedCircuitVoltagePowerSupply">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>301</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Напряжение замкнутой цепи блока питания</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnVoltageOnTheHousing">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Пуск</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnInsulationResistance">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Пуск</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenCircuitVoltageGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Пуск</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClosedCircuitVoltageGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Пуск</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClosedCircuitVoltageBattery">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>120</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Пуск</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnInsulationResistanceMeasuringBoardUUTBB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>140</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Пуск</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenCircuitVoltagePowerSupply">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>160</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Пуск</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClosedCircuitVoltagePowerSupply">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>180</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Пуск</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbDepassivation">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>301</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Распассивация</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDepassivation">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Пуск</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenCircuitVoltageGroup_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Продолжить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnVoltageOnTheHousing_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Продолжить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClosedCircuitVoltagePowerSupply_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>180</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Продолжить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClosedCircuitVoltageGroup_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Продолжить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnInsulationResistance_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Продолжить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenCircuitVoltagePowerSupply_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>160</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Продолжить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnInsulationResistanceMeasuringBoardUUTBB_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>140</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Продолжить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDepassivation_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>100</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Продолжить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClosedCircuitVoltageBattery_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>120</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Продолжить</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>451</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxDiagnosticDevice">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>471</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Диагностируемое устройство</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Батарея:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxBatteryList">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>9ER20P-20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9ER20P-28</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9ER14P-24</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9ER14PS-24</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbIsUUTBB">
        <property name="text">
         <string>УУТББ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbIsImitator">
        <property name="text">
         <string>Имитатор</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCheckConnectedBattery">
        <property name="text">
         <string>Проверить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Дата производства:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEditBatteryBuild"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Номер батареи:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditBatteryNumber"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnStopAutoModeDiagnostic">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>470</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Стоп</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="menuExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="menuCOMPort">
   <property name="text">
    <string>Порт</string>
   </property>
  </action>
  <action name="menuReset">
   <property name="text">
    <string>Сброс</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Сформировать файл-отчет</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnEventLogClear</sender>
   <signal>clicked()</signal>
   <receiver>EventLog</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>762</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>592</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
